client/src/components/overview/overview-components/Price.jsx

```diff
@@ -6,9 +6,18 @@ const StyledOverviewPrice = styled.div`
 `;
 
 export default function OverViewPrice({ price }) {
-  return (
-    <StyledOverviewPrice>
-      <p>${price.original_price}</p>
-    </StyledOverviewPrice>
-  );
+  if (price.sale_price) {
+    return (
+      <StyledOverviewPrice>
+        <span style={{textDecoration: 'line-through', color: 'red'}}>${price.original_price}</span><br></br>
```

Comment:
I believe it is typically the sale price that appears in red, not the original price. Perhaps you can coordinate with Chris so that you both style the pricing for sales the same.